x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/date_histogram.tsx

```diff
@@ -78,7 +82,8 @@ export const dateHistogramOperation: OperationDefinition<DateHistogramIndexPatte
     };
   },
   isTransferable: (column, newIndexPattern) => {
-    const newField = newIndexPattern.fields.find((field) => field.name === column.sourceField);
+    const c = column as DateHistogramIndexPatternColumn;
```

Comment:
It seems a bit ugly we need those casts everywhere. What was the reason for typing `column` by the column type of the current operation?